web/src/features/hfiCalculator/components/HFIPageSubHeader.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+
+import { Button, FormControl, Grid } from '@material-ui/core'
+import FireCentreDropdown from 'features/hfiCalculator/components/FireCentreDropdown'
+import { isUndefined } from 'lodash'
+import { FireCentre } from 'api/hfiCalcAPI'
+import AboutDataModal from 'features/hfiCalculator/components/AboutDataModal'
+import { HelpOutlineOutlined } from '@material-ui/icons'
+import DatePicker from 'components/DatePicker'
+import { formControlStyles } from 'app/theme'
+
+const useStyles = makeStyles(theme => ({
+  ...formControlStyles,
+  root: {
+    minHeight: 50,
+    maxHeight: 62,
+    marginBottom: '1rem',
+    paddingBottom: '0.25rem',
+    paddingTop: '0.25rem',
+    paddingLeft: '1rem',
+    paddingRight: '1rem',
+    fontSize: '1.3rem',
+    background: theme.palette.primary.light,
+    color: theme.palette.primary.contrastText,
+    alignContent: 'center'
+  },
+  gridContainer: {
+    height: '85%'
+  },
+  positionStyler: {
```

Comment:
I think this class is unused and can be deleted

Reply:
Good eye. Deleted